code/modules/spells/ability_types/realized.dm

```diff
@@ -93,3 +93,43 @@
 	desc = "Your sinful actions have made your soul more vulnerable to PALE attacks."
 	icon = 'icons/mob/actions/actions_ability.dmi'
 	icon_state = "judgement"
+
+/obj/effect/proc_holder/ability/fire_explosion
+	name = "Match flame"
+	desc = "An ability that deals high amount of RED damage to EVERYONE around the user after short delay."
+	action_icon_state = "fire0"
+	base_icon_state = "fire"
+	cooldown_time = 30 SECONDS
+	var/explosion_damage = 1000 // Humans receive half of it.
```

Comment:
Only concern is that this may be a lot? I'd like to discuss how we should balance Realization EGOs and their damages at a later point

Reply:
I too, thought that it may be a lot, but unlike other realizations, this one doesn't have any "auxiliary" effects.
1000 damage every 30 seconds after a formidably long delay that can also damage your team is balanced, in my opinion.